lib/livebook_web/live/layout_helpers.ex

```diff
@@ -247,7 +253,7 @@ defmodule LivebookWeb.LayoutHelpers do
       "h-7 flex items-center hover:text-white #{text_color} border-l-4 #{border_color} hover:border-white"
 
     if tooltip = Provider.connection_error(hub) do
-      [to: to, data_tooltip: tooltip, class: "tooltip right " <> class]
+      [to: to, href: to, data_tooltip: tooltip, class: "tooltip right " <> class]
```

Comment:
May have missed when changing to `<.link>`, I think it should be this:

```suggestion
      [navigate: to, data_tooltip: tooltip, class: "tooltip right " <> class]
```